modules/localgov_campaigns_paragraphs/js/localgov-tabs.js

```diff
@@ -0,0 +1,227 @@
+
+            // Add initialised class.
+            tabs.classList.add(tabsInitialisedClass);
+          }();
+
+          if ((typeof _ret === 'undefined' ? 'undefined' : _typeof(_ret)) === "object") return _ret.v;
+        }
+      };
+
+      var destroy = function destroy() {
+        if (tabs.classList.contains(tabsInitialisedClass)) {
+          // Remove tabs.
+          tabs.querySelectorAll('.tabs__nav')[0].remove();
+
+          // Remove attributes from tab panels.
+          for (var i = 0; i < tabPanels.length; i++) {
+            tabPanels[i].removeAttribute('role');
+            tabPanels[i].removeAttribute('tabindex');
+            tabPanels[i].removeAttribute('aria-labelledby');
+            tabPanels[i].removeAttribute('id');
+
+            if (tabs.querySelectorAll('.tab-panel--active').length > 0) {
+              tabs.querySelectorAll('.tab-panel--active')[0].classList
+                .remove('tab-panel--active');
+            }
+          }
+
+          // Remove init class.
+          tabs.classList.remove(tabsInitialisedClass);
+        }
+      };
+
+      var breakpointCheck = function breakpointCheck() {
+        if (mq.matches) {
+          destroy();
+        } else {
+          create();
+        }
+      };
+
+      // Trigger create/destroy functions at different screen widths
+      // based on the value of data-accordion-tabs-switch attribute.
+      if (window.matchMedia) {
+        var mq = window.matchMedia(`(max-width: ${breakpoint})`);
+        mq.addListener(function (e) {
+          breakpointCheck();
+        });
+        breakpointCheck();
+      }
+
+    },
+
+    /**
+     * Switch tab.
+     *
+     * @param {HTMLElement} newActiveTab
+     *   Tab element.
+     * @param {HTMLElement} tabs
+     *   Tabs element.
+     */
+    switchTab: function switchTab(newActiveTab, tabs) {
+      var newActivePanelId = newActiveTab.getAttribute('aria-controls');
+      var newActivePanel = tabs.querySelectorAll('#' + newActivePanelId);
+      var activePanelClass = 'tab-panel--active';
+      var oldActiveTab = tabs.querySelectorAll('.tabs__controls [aria-selected="true"]').length > 0 ? tabs.querySelectorAll('.tabs__controls [aria-selected="true"]')[0] : null;
+
+      // Deactivate current active control.
+      if (oldActiveTab) {
+        oldActiveTab.setAttribute('aria-selected', false);
+        oldActiveTab.setAttribute('tabindex', '-1');
+      }
+
+      // Set new active control.
+      newActiveTab.setAttribute('aria-selected', true);
+      newActiveTab.removeAttribute('tabindex');
+
+      // Deactivate current active panel.
+      if (tabs.querySelectorAll('.' + activePanelClass).length > 0) {
+        tabs.querySelectorAll('.' + activePanelClass)[0].classList
+          .remove(activePanelClass);
+      }
+
+      // Set new active panel.
+      newActivePanel[0].classList.add(activePanelClass);
+      if (oldActiveTab) {
+        newActivePanel[0].focus();
+      }
+    }
+  };
+})();
```

Comment:
Usually, this is where you would pass in the "Drupal" object and then accept it as a function parameter above.  Drupal core has some good [examples](https://git.drupalcode.org/project/drupal/-/blob/9.0.x/core/misc/checkbox.js).

Reply:
Added

Reply:
Thanks :)